lib/cct/cloud/control_node.rb

```diff
@@ -118,6 +119,14 @@ def load! force: false
       @loaded = true
     end
 
+    def write_conf_file (filename, section, content)
+      ini_content = {
+        "section #{section}" => content
+      }
+      conf_file = IniFile.new(:filename => filename, :content => ini_content)
```

Comment:
thats on the admin node ,not on the control node.. you'd have to do that via ssh ..